src/number-runtime.js

```diff
@@ -0,0 +1,57 @@
+define([
```

Comment:
All the `src/*-runtime.js` follow the same patterns. The `format/parseX` methods validate parameters, the others map arguments through `runtimeKey`. Is there any way to get rid of these and generate them in the build or with the compiler?